drivers/usb/host/xhci-pci.c

```diff
@@ -62,7 +62,7 @@
 #define PCI_DEVICE_ID_INTEL_MAPLE_RIDGE_XHCI		0x1138
 #define PCI_DEVICE_ID_INTEL_ALDER_LAKE_PCH_XHCI		0x51ed
 #define PCI_DEVICE_ID_INTEL_ALDER_LAKE_N_PCH_XHCI	0x54ed
-#define PCI_DEVICE_ID_INTEL_RAPTOR_LAKE_XHCI 0xa71e
+#define PCI_DEVICE_ID_INTEL_RAPTOR_LAKE_XHCI            0xa71e
```

Comment:
How about MBL and PTL?

Reply:
MBL and RPL has the same XHCI controller model.
PTL XHCI role switch is different from RPL, and not enabled yet. (IA enabled them in UEFI BIOS)